### ISSUE TITLE
Reporting bug and temporary solution for synonyms

### ISSUE THREAD

**Issue by ZainW** [ankane/searchkick#913]
So part of my project is to search by proper names such as Arc Warden, all fo the searching functions work for it but if i want to write a synonym for it I cannot do `[["Arc Warden", "aw"]]` I must do `[["arcwarden", "aw"]]`

is it possible to trim spaces when searchkick imports the synonym list from CSV for example? I would like to keep my synonyms list as small and proper as possible

I will test with the capital letters once 2.2.2 has been released

tldr: dont write synonyms with spaces, remove them.

If synonyms are only meant for individual words that is ok, just reporting for future people who may run into the same issue as me.

**Comment by pedrorio** [comment]
Can someone else replicate this? Is this solved?

**Comment by ankane** [comment]
Hey @ZainW, thanks for reporting 👍  I updated to remove spaces automatically, and added a note about synonyms needing to be two words or less.
